packages/lodestar-validator/src/services/attestation.ts

```diff
@@ -47,6 +48,8 @@ export class AttestationService {
   private readonly logger: ILogger;
 
   private nextAttesterDuties: Map<Slot, IAttesterDuty[]> = new Map();
+  private aggregatorTimeout?: NodeJS.Timeout;
+  private controller: AbortController = new AbortController();
```

Comment:
might make more sense to initialize in on start (soo we can start-stop multiple times)